apps/server/src/modules/user/legacy/repo/helper/search-query-helper.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { ObjectId } from 'bson';
+import { UserSearchQuery } from '../../interfaces';
+import { SearchQueryHelper } from '.';
+import { UsersSearchQueryParams } from '../../controller/dto';
+
+describe('Search query helper', () => {
+	afterEach(() => {
+		jest.resetAllMocks();
+	});
+
+	describe('when searching by searchQuery', () => {
```

Comment:
This test is only for the setSearchParametersIfExist method, which should be in a describe. And a test for the setDateParametersIfExists method is missing.